packages/core/lib/queues/AbstractQueueService.ts

```diff
@@ -248,6 +274,11 @@ export abstract class AbstractQueueService<
         ? // @ts-ignore
           message[this.messageTypeField]
         : undefined
+    const messageDeduplicationId =
```

Comment:
to reduce probability of errors, if (consumer)publisherDeduplication is enabled, and deduplicationId field is not present, I'd throw an error